test/spec/native_spec.js

```diff
@@ -97,8 +97,28 @@ const ortbBid = {
       },
       privacy: 'https://privacy-link.example',
       ver: '1.2'
-    }
-  },
+    },
+    // Native bid response always still have prebid legacy fields
+    title: 'Native Creative',
```

Comment:
with this change the tests using `ortbBid` are no longer real tests - they are satisfied from their precondition. 

I think the correct thing to do is to refactor them to test `toLegacyResponse` instead of the `getAssets` methods, now that the former is exposed. The logic should be very similar (naked ORTB response comes in, legacy response comes out).